lib/src/inline_parser.dart

```diff
@@ -871,9 +875,19 @@ class DelimiterRun implements Delimiter {
       'canClose: $canClose>';
 }
 
+class DelimiterTag {
+  DelimiterTag(this.tag, this.indicators);
+
+  // Tag name of AST Element
```

Comment:
Grammar nit: "Tag name of the HTML element." (ending with a period)

Reply:
Fixed